script.js

```diff
@@ -4,3 +4,103 @@
 // - A User should only be able to select one box per turn
 // - A User should not be able to select a box that is already selected
 // - When game is over (whether a win or tie) the users should be prompted to play again and the game should be reset
+
+let gamePiece = "";
+let playerTurn = true;
+let text = document.getElementById("text");
+const xGamePiece = "X";
+const oGamePiece = "O";
+const gameSpaces = document.querySelectorAll(".gameSpace");
+
+gameSpaces.forEach(addEventListenerToGameSpaces);
+
+function addEventListenerToGameSpaces(gameSpace) {
+  gameSpace.addEventListener("click", useGamePiece);
+}
+
+text.innerHTML += "Player X turn";
+
+function checkWin() {
+  if (
+    gameSpaces[0].innerHTML === gameSpaces[1].innerHTML &&
+    gameSpaces[1].innerHTML === gameSpaces[2].innerHTML &&
+    gameSpaces[0].innerHTML !== ""
+  ) {
+    return true;
+  }
+
+  if (
+    gameSpaces[3].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[5].innerHTML &&
+    gameSpaces[3].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[6].innerHTML === gameSpaces[7].innerHTML &&
+    gameSpaces[7].innerHTML === gameSpaces[8].innerHTML &&
+    gameSpaces[6].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[0].innerHTML === gameSpaces[3].innerHTML &&
+    gameSpaces[3].innerHTML === gameSpaces[6].innerHTML &&
+    gameSpaces[0].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[1].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[7].innerHTML &&
+    gameSpaces[1].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[2].innerHTML === gameSpaces[5].innerHTML &&
+    gameSpaces[5].innerHTML === gameSpaces[8].innerHTML &&
+    gameSpaces[2].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[0].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[8].innerHTML &&
+    gameSpaces[0].innerHTML !== ""
+  ) {
+    return true;
+  }
+  if (
+    gameSpaces[2].innerHTML === gameSpaces[4].innerHTML &&
+    gameSpaces[4].innerHTML === gameSpaces[6].innerHTML &&
+    gameSpaces[2].innerHTML !== ""
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function useGamePiece(event) {
+  if (playerTurn === true) {
```

Comment:
you can just say
`if(playerturn){
   ...
 } else ...`
 since `playerTurn` is a boolean it can only be `true` or `false`

Reply:
in fact you could also use a ternary operator for this as well to simplify it even further
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator